apps/easypid/src/features/menu/FunkeAboutScreen.tsx

```diff
@@ -37,14 +39,17 @@ export function FunkeAboutScreen() {
             </Paragraph>
           </Paragraph>
         </YStack>
-        <YStack gap="$2">
+        <YStack gap="$2" fg={1} jc="space-between">
           <InfoButton
             variant="view"
             title="Privacy Policy"
             description="Open the privacy policy"
             routingType="external"
             onPress={openPrivacyPolicy}
           />
+          <Paragraph py="$4" mx="auto" variant="sub" fontSize={13} fontWeight="$medium">
+            EasyPID version: {pj.version}
```

Comment:
You can use expo-constants (already installed) and it also have the build number (which increaes every release but not the package json version)

Reply:
https://docs.expo.dev/versions/latest/sdk/constants/#iosmanifest  
https://docs.expo.dev/versions/latest/sdk/constants/#androidmanifest